documentation/src/docs/asciidoc/docinfos/index-docinfo.html

```diff
@@ -0,0 +1 @@
+<link rel="icon" type="image/png" href="{{ site.baseurl }}/img/junit5-logo.png">
```

Comment:
If the favicon is shown for you locally, that's probably your browser fooling you. Chrome, for example, caches the favicon for a domain quite agressively.

As you can see from the screenshot below, `{{ site.baseurl }}` is not replaced by Jekyll:
 
<img width="1085" alt="screen shot 2017-05-17 at 15 09 06" src="https://cloud.githubusercontent.com/assets/214207/26155276/ea6aadb2-3b12-11e7-8996-ab88612cb810.png">

In addition, even if this did work, the Javadoc pages would not benefit from the current changes.